docs/_basic/listening_actions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Listening to actions
+lang: en
+slug: action-listening
+order: 5
+---
+
+<div class="section-content">
+Your app can listen to user actions like button clicks, and menu selects, using the `action` method.
+
+Actions can be filtered on an `action_id` of type `str` or `RegExp` object. `action_id`s act as unique identifiers for interactive components on the Slack platform.
```

Comment:
In all pages, `RegExp` should be replaced with `re.Pattern`. Python doesn't have the type `RegExp` unlike JS.

Reply:
I just now noticed that in Python, they leave off the `p` when referring to `RegEx`. 🤦‍♀️ 

Regardless, switched these out for `re.Pattern`.